pkpdapp/pkpdapp/migrations/0005_auto_20210805_0831.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.1.9 on 2021-08-05 08:31
```

Comment:
Can you please fold this automatically generated migration into the initial migration, the easiest way to do this is:
1. delete the initial migration, and move all the custom migration files out of the migrations folder
2. delete your database
3. run `python manage.py makemigration`, this will create the initial migration file
4. movve all the custom migration files back to the migrations folder
5. run `python manage.py migrate` to run all the initial and custom migration files

Reply:
@martinjrobins I'm not sure what exactly are the files you are talking about in the first step. When I delete/move them and run the commands again they come back and are exactly the same
Is it why there are a lot of "django.db.utils.OperationalError: table "pkpdapp_variable" already exists" errors in the pull request tests ?